tools/ota_e2e_tests/aws_ota_test/aws_ota_project.py

```diff
@@ -134,6 +136,7 @@ def buildProject(self):
             print('====> Command run completed with the return code: ', proc.returncode)
             returnCodes.append(proc.returncode)
 
+        sleep(5)
```

Comment:
What is this sleep for? Is to give the process time to generate binaries after returning?

Reply:
Yeah .. It is causing the test to fail saying could not find output binary even after its actually present. Just added sleep to test. If this works i'll add proper handling for this scenario.

Reply:
Maybe we can add a comment if you plan to keep it?

Reply:
Added the comment in the below commit :-)